kasa/smart/modules/lightstripeffect.py

```diff
@@ -53,6 +52,28 @@ def effect(self) -> str:
             return name
         return self.LIGHT_EFFECTS_OFF
 
+    @property
+    def is_active(self) -> bool:
+        """Return if effect is active."""
+        eff = self.data["lighting_effect"]
+        # softAP has enable=1, but brightness 0 which fails on tests
+        return bool(eff["enable"]) and eff["name"] in self._effect_list
+
+    @property
+    def brightness(self) -> int:
+        """Return effect brightness."""
+        eff = self.data["lighting_effect"]
+        return eff["brightness"]
+
+    async def set_brightness(self, brightness: int, *, transition: int | None = None):
+        """Set effect brightness."""
+        if brightness <= 0:
+            return await self.set_effect(self.LIGHT_EFFECTS_OFF)
+
+        # Need to pass bAdjusted to keep the existing effect running
+        eff = {"brightness": brightness, "bAdjusted": True}
+        return await self.set_custom_effect(eff)
```

Comment:
This seems like an important thing to have covered by tests. Could we put a `pytest.skip` in the test if it `has_custom_effects` is false and a quick line to test this? Happy to do this later if it's a big deal getting the test framework to support all the custom effect logic. Priority is to get the fix done obviously.

Reply:
Agreed, I'll add test for this logic :+1:

Reply:
We have separate tests for light_strip_effect alongside the common tests.